### ISSUE TITLE
No right autocorrects

### ISSUE THREAD

**Issue by wbotelhos** [backus/rubocop-rspec#518]
Hi,

I'm having some strange corrections:

Before:

```
# frozen_string_literal: true

require 'common_helper'
require 'support/capybara_box'
require 'support/database_cleaner'
require 'support/factory_bot'

RSpec.describe Dashboard do
  before do
    page.set_rack_session(
      current_grades_ids: [
        1
      ],

      current_student_id: student.id
    )
  end
end
```

After:

```
# frozen_string_literal: true

require 'common_helper'
require 'support/capybara_box'
support/database_cleaner'
require 'support/factory_bot'

RSpec.describe Dashboard do
  before do
    page.set_rack_session(
      current_grades_ids: [
        1
      ],

      current_student_id: student.id
    )
  end
end
```

or

before:

```
# frozen_string_literal: true

require 'common_helper'
require 'support/capybara_box'
require 'support/database_cleaner'
require 'support/factory_bot'
require 'support/shared/enroll_situation'

RSpec.describe Dashboard, '#user:choose_schedule', :js do
  before do
    page.set_rack_session(
      current_grades_ids: [
        'xxx',
        'xxx',
        'xxx',
        'xxx'
      ],

      current_student_id: student.id,
      current_user_id:    student.user_id
    )
  end

  it 'works' do
  end
end
```

After:

```
# frozen_string_literal: true

require 'common_helper'
require 'support/capybara_box'
require 'support/database_cleaner'
require 'support/factory_bot'
equire 'support/shared/enroll_situation'
  .describe Dashboard, '#user:choose_schedule', :js do
  before do
    page.set_rack_session(
      current_grades_ids: %w[
        xxx
        xxx
        xxx
        xxx
      ],

      current_student_id: student.id,
      current_user_id:    student.user_id
    )
  end

  it 'works' do
  end
end
```

Thanks for this great project. (:

**Comment by Darhazer** [comment]
It seems that those auto-corrections are done by [rubocop](https://github.com/bbatsov/rubocop) itself, and not by rubocop-rspec

**Comment by wbotelhos** [comment]
Ok @Darhazer ,

I'll open the issue there.

Thanks.

**Comment by Darhazer** [comment]
Both are fixed in rubocop 0.52.1
